src/Router.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import ErrorPage from './components/ErrorPage';
+import Navbar from './components/Navbar';
+import Books from './components/Books';
+import Categories from './components/Categories';
+
+function RouterNav() {
+  return (
+    <>
+      <BrowserRouter className="app-container">
+        <Navbar />
+        <Routes>
+          <Route exact path="books" element={<Books />} />
+          <Route path="categories" element={<Categories />} />
+          <Route path="*" element={<ErrorPage />} />
+        </Routes>
```

Comment:
- [ ] Actually, the requirement was to make 2 pages but there is no harm if you create one more page. The issue is that your Books page should be your homepage, not the ErrorPage :+1: to follow the requirements :blush:

You can do that by following Routes below :arrow_down:

```jsx
      <Routes>
          <Route path="/" element={<Home />} />
          <Route path="/categories" element={<Categories />} />
        </Route>
      </Routes>

```